pkg/api/converterfromosaapi.go

```diff
@@ -6,8 +6,8 @@ import (
 
 // ConvertVLabsOpenShiftClusterToContainerService converts from a
 // v1.OpenShiftCluster to a ContainerService.
-func ConvertVLabsOpenShiftClusterToContainerService(oc *v1.OpenShiftCluster) *ContainerService {
-	cs := &ContainerService{
+func ConvertVLabsOpenShiftClusterToContainerService(oc *v1.OpenShiftManagedCluster) *OpenShiftCluster {
```

Comment:
Any reason why these don't use the same name?

Reply:
fixed and squashed